client/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM node:5.9
```

Comment:
Yeah, that's what I'm figuring, honestly.  It's good to push your minor every so often (and obviously just pick whatever the latest patch is, hence the lack of a patch number in this tag) but 5.9->5.10 isn't worthwhile